### DISCUSSION TITLE
Install on Pi5?

### DISCUSSION THREAD

**Discussion by Tiziano268** [Screenly/Anthias#2046] [General]
I am new here. How can I install it to a Pi5?

**Comment by nicomiguelino** [comment]
Anthias doesn't support Pi 5 at the moment, but open-source contributions are always welcome.

**Comment by nicomiguelino** [comment]
@Tiziano268, we recently added support for Raspberry Pi 5. Please head over to [this documentation](https://github.com/Screenly/Anthias/blob/master/docs/installation-options.md) for details about the installation options.

**Comment by thassani2026** [comment]
Hi,
Looking to the documentation, there is option to install anthisa OS. i can't found it under Imager v2.0.6. Just note that i'm using Pi 5.

Can some one help ? thank you

**Reply by nicomiguelino** [reply]
@thassani2026, if you want to install Anthias via a disk image, you can download the latest version by going to [this link](https://hub.balena.io/fleets-for-good/2209774/anthias-pi5).
- Once you're in the page, click **Join**.
- A dialog box will appear. Click the dropdown beside the **Flash** button (located on the lower left part of the dialog).
- Click **Download balenaOS**.

Another option is to install Anthias via a command-line installler. More details could be found [here](https://github.com/Screenly/Anthias/blob/master/docs/installation-options.md#installing-on-raspberry-pi-os-lite-or-debian).

I hope that helps. Please let me know if you need further help. Thank you.

**Comment by thassani2026** [comment]
Thank you, all working fine.

Just need to config a WiFi extender. Any link ?


________________________________
De : Nico Miguelino ***@***.***>
Envoyé : 20 février 2026 09:50
À : Screenly/Anthias ***@***.***>
Cc : Tarik Hassani ***@***.***>; Mention ***@***.***>
Objet : Re: [Screenly/Anthias] Install on Pi5? (Discussion #2046)

Vous n’obtenez pas souvent d’e-mail à partir de ***@***.*** Pourquoi c’est important<https://aka.ms/LearnAboutSenderIdentification>

@thassani2026<https://github.com/thassani2026>, if you want to install Anthias via a disk image, you can download the latest version by going to this link<https://hub.balena.io/fleets-for-good/2209774/anthias-pi5>.

  *   Once you're in the page, click Join.
  *   A dialog box will appear. Click the dropdown beside the Flash button (located on the lower left part of the dialog).
  *   Click Download balenaOS.

Another option is to install Anthias via a command-line installler. More details could be found here<https://github.com/Screenly/Anthias/blob/master/docs/installation-options.md#installing-on-raspberry-pi-os-lite-or-debian>.

I hope that helps. Please let me know if you need further help. Thank you.

**Reply by nicomiguelino** [reply]
If your Wi-Fi extender has an Ethernet port in it, I would recommend connecting your Raspberry Pi into the range extender via an Ethernet cable.

If that won't be possible, I would suggest configuring the image before downloading it.
Once the dialog appears after you click **Join**, select **Wifi + Ethernet**.

<img width="1222" height="759" alt="image" src="https://github.com/user-attachments/assets/d359b4e5-a095-43f1-b8c8-369d4d9a720b" />

Please fill in the **Wifi SSID** and the **Wifi Passphrase**. Once ready, you can now Either **Flash** or **Download** the balenaOS image. After that, you can re-flash your micro-SD card.

**Comment by vpetersson** [comment]
Raspberry Pi 5 is now fully supported, and you confirmed it is working for you. Closing as resolved.
